gcn/notices/icecube/single_neutrino_alerts.schema.json

```
@@ -0,0 +1,19 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/icecube/single_neutrino_alerts.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "IceCubeSingleNeutrinoAlert",
```

```suggestion
  "title": "IceCube Single Neutrino Alert",
```
Why not to make it easy to read, as any string can be added?

---

thanks, updated.